### DISCUSSION TITLE
Is --proxy-headers applied in the uvicorn CMD in the image?

### DISCUSSION THREAD

**Discussion by PhaniGudepu-Stoik** [tiangolo/uvicorn-gunicorn-fastapi-docker#329] [Questions] [answered]
I am using python3.11-2023-04-03 image. and wanted to know if "--proxy-headers" is attached while starting the server

**Comment by sy6sy2** [comment]
Same question here.
I guess the answer is no because I have this issue with this docker image https://stackoverflow.com/questions/76672798/fastapi-starlette-nginx-proxy-url-scheme-in-request-object-not-correct

**Comment by tiangolo** [comment, marked as answer]
I think nope, because it depended on what is available to Gunicorn, and that can also be passed to the Uvicorn workers.

On the other side, now that Uvicorn supports managing workers with `--workers`, including restarting dead ones, there's no need for Gunicorn. That also means that it's much simpler to build a Docker image from scratch now, I updated the docs to explain it.

Because of that, I deprecated this Docker image: https://github.com/tiangolo/uvicorn-gunicorn-fastapi-docker#-warning-you-probably-dont-need-this-docker-image
